src/content/ai/create-with-ai.md

```diff
@@ -78,18 +97,86 @@ and fix errors, hot reload, get the selected widget, and more. This bridges
 the gap between the AI's natural language understanding, and
 Dart and Flutter's suite of developer tools.
 
-To get started, check out the official documentation for the [Dart and Flutter MCP server][dart-mcp-dart-docs]
+To get started, check out the official documentation for the
+[Dart and Flutter MCP server][dart-mcp-dart-docs]
 on dart.dev and the [Dart and Flutter MCP repository][dart-mcp-github].
 
----
-
-[Gemini Code Assist]: https://codeassist.google/
-[Gemini CLI]: {{site.github}}/google-gemini/gemini-cli
-[Gemini CLI codelab]: https://codelabs.developers.google.com/gemini-cli-hands-on
-[Firebase AI Logic]: {{site.firebase}}/docs/ai-logic
-[firebase-ai-logic-docs]: {{site.firebase}}/docs/ai-logic/get-started
-[How to build agentic apps with Flutter and Firebase AI Logic]: {{site.yt.watch}}/watch?v=xo271p-Fl_4
-[Flutter AI Toolkit]: {{site.url}}/ai-toolkit
 [dart-mcp-dart-docs]: {{site.dart-site}}/tools/mcp-server
 [dart-mcp-github]: {{site.github}}/dart-lang/ai/tree/main/pkgs/dart_mcp_server
 [dart-mcp-flutter-docs]: #dart-and-flutter-mcp-server
+
+## Rules
```

Comment:
I'd love to ground this in Rules for Flutter (and Dart?)

Reply:
Sounds great. Updating!